src/components/App/App.css

```diff
@@ -1,6 +1,17 @@
 /* MOBILE-FIRST VIEW */
 .App {
   text-align: center;
+  display: flex;
+  flex-direction: column;
+  background-color: #3D157A;
```

Comment:
In my next PR, for the ErrorPage component, I'm also including a fix here for the background color (with gradient) and fixing the height to 100vh. Just mentioning it because I can see that the purple background isn't quite filling the screen in this PR, but I think that's fine since my next one will fix it.

Reply:
Sounds good, thanks for catching this!